### DISCUSSION TITLE
Is there a way to bridge the vm?

### DISCUSSION THREAD

**Discussion by Opswatch** [smartxworks/virtink#95] [Q&A]
Is there a way to bridge the vm to the network of the host node so we can run something like ansible against it instead of having to use kubectl for ssh?

**Comment by fengye87** [comment]
You should be able to access Virtink VMs like the way you access Pods from the host node. We use kubectl in the demo just to make it more universal.

**Reply by daxiguafan** [reply]
Use ingress to expose the ssh service?

**Reply by fengye87** [reply]
No problem, it should work.
